app/views/comparisons/change_in_electricity_holiday_consumption_previous_holiday/_tables.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'comparisons/change_in_electricity_holiday_consumption_previous_years_holiday/tables', results: results %>
```

Comment:
If there are tables with same layout, can we move the shared partials to a common directory. E.g. `comparisons/tables/change_in_electricity_holiday_consumption`? We're likely to have the exact same report for gas and storage heater too, so worth extracting a single common table if we can.